environment.yml

```diff
@@ -24,6 +24,7 @@ channels:
 
 dependencies:
     #- python=3.7.6
+    - cpuonly
```

Comment:
why is this needed?

Reply:
`PyTorch & Conda` CI is installing gpu version of pytorch whereas we are only testing in cpu mode, right?
so, adding that (installing cpu version of pytorch only) will save a few minutes of downloading and installing pkgs

Reply:
but the env shall be generic not just for local CPU testing and we assume that most users have GPU, so if this is needed let's add it in the CI not here

Reply:
Okay I will apply the suggestions once #3383 is merged